cwl/tools/copy_number_consensus_call.cwl

```diff
@@ -13,14 +12,29 @@ arguments:
   - position: 0
     shellQuote: false
     valueFrom: >-
-      -p scratch && mkdir results && python /d3b-bixu-data-assembly/scripts/copy_number_consensus_call/scripts/merged_to_individual_files.py --manta $(inputs.input_formatted_mantaSV.path)  --cnvkit $(inputs.input_formatted_cnvkit.path) --freec $(inputs.input_formatted_controlfreeC.path) --snake ./scratch/config_snakemake.yaml --scratch /scratch --uncalled ./results/uncalled_samples.tsv && snakemake -s /d3b-bixu-data-assembly/scripts/copy_number_consensus_call/Snakefile --configfile /scratch/config_snakemake.yaml -j --restart-times 2 && cp ./results/data-cnv-consensus.seg.gz .
+      -p scratch &&
+      mkdir results &&
+      python /d3b-bixu-data-assembly/scripts/copy_number_consensus_call/scripts/merged_to_individual_files.py 
+      --cnvkit $(inputs.input_formatted_cnvkit.path)
+      --freec $(inputs.input_formatted_controlfreeC.path)
+      --snake  ./scratch/config_snakemake.yaml 
+      --scratch ./scratch 
+      --uncalled ./results/uncalled_samples.tsv
+      $(inputs.run_WGS_or_WXS == 'WGS' ? '--manta ' + inputs.input_formatted_mantaSV.path : '')
+      && snakemake -s /d3b-bixu-data-assembly/scripts/copy_number_consensus_call/$(inputs.run_WGS_or_WXS == 'WGS' ? 'Snakefile' : 'Snakefile_WXS') 
+      --configfile ./scratch/config_snakemake.yaml 
```

Comment:
Looking at the test logs: 
```
2021-08-03T14:05:15.336718817Z Relative file path './scratch/endpoints/all_CNVs_combined.tsv' starts with './'. This is redundant and strongly discouraged. It can also lead to inconsistent results of the file-matching approach used by Snakemake. You can simply omit the './' for relative file paths.
```
Should you drop the `./` notation? Seems to be annoying snakemake.

Reply:
done ,fixed : https://cavatica.sbgenomics.com/u/zhangb1/data-assembly-dev/tasks/510d2431-b51f-41ea-9227-ddbb81992b79/stats/